appveyor.yml

```diff
@@ -23,22 +23,51 @@ environment:
       EXAMPLE: 01-executable
       CONFIG: Release
 
+    - TOOLCHAIN: "vs-14-2015"
+      EXAMPLE: 02-library
+      CONFIG: Release
+
+    - TOOLCHAIN: "vs-14-2015"
+      EXAMPLE: 03-shared-link
+      CONFIG: Release
+
     - TOOLCHAIN: "mingw"
       EXAMPLE: 01-executable
       CONFIG: Release
 
-    - TOOLCHAIN: "msys"
+    - TOOLCHAIN: "mingw-cxx14"
       EXAMPLE: 01-executable
       CONFIG: Release
+      CMAKE_MAKE_PROGRAM: mingw32-make.exe
+      MINGW_PATH: C:\mingw-w64\x86_64-7.2.0-posix-seh-rt_v5-rev1\mingw64\bin
+      ADD_TO_PATH: C:\mingw-w64\x86_64-7.2.0-posix-seh-rt_v5-rev1\mingw64\bin 
 
-    - TOOLCHAIN: "vs-14-2015"
-      EXAMPLE: 02-library
+    - TOOLCHAIN: "mingw-cxx17"
+      EXAMPLE: 01-executable
       CONFIG: Release
+      CMAKE_MAKE_PROGRAM: mingw32-make.exe
+      MINGW_PATH: C:\mingw-w64\x86_64-7.2.0-posix-seh-rt_v5-rev1\mingw64\bin
+      ADD_TO_PATH: C:\mingw-w64\x86_64-7.2.0-posix-seh-rt_v5-rev1\mingw64\bin 
 
-    - TOOLCHAIN: "vs-14-2015"
-      EXAMPLE: 03-shared-link
+    - TOOLCHAIN: "msys"
+      EXAMPLE: 01-executable
       CONFIG: Release
 
+    #- TOOLCHAIN: "msys-cxx14"
```

Comment:
Please remove unused code.

Reply:
Sorry, I was experimenting with fixing msys.  I'll do this work in another branch so you won't get nagged by changes, but I'll remove anything I can't fix.